.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -129,26 +139,65 @@ def str2bool(v):
     base_branch=args.base_branch,
 )
 
+
+def _load_results() -> list:
+    '''
+    Load accumulated notebook test results
```

Comment:
Add comment about returning empty array if the result can't be loaded.

Reply:
done

Reply:
Did you push?